mod/settings.php

```diff
@@ -150,20 +148,19 @@ function settings_post(App $a) {
 	}
 
 	if (($a->argc > 2) && ($a->argv[1] === 'oauth')  && ($a->argv[2] === 'edit'||($a->argv[2] === 'add')) && x($_POST, 'submit')) {
-
 		check_form_security_token_redirectOnErr('/settings/oauth', 'settings_oauth');
 
-		$name   	= ((x($_POST, 'name')) ? $_POST['name'] : '');
-		$key		= ((x($_POST, 'key')) ? $_POST['key'] : '');
-		$secret		= ((x($_POST, 'secret')) ? $_POST['secret'] : '');
-		$redirect	= ((x($_POST, 'redirect')) ? $_POST['redirect'] : '');
-		$icon		= ((x($_POST, 'icon')) ? $_POST['icon'] : '');
-		if ($name=="" || $key=="" || $secret=="") {
-			notice(t("Missing some important data!"));
+		$name     = defaults($_POST, 'name'    , '');
```

Comment:
I have the feeling that I don't really that free floating comma and would prefer the comma directly behind the parameter instead (and much spaced behind the comma - not before)

Reply:
My reasoning is that the recurring delimiter punctuations should be aligned so that the `$key` parameter can be read as easily as possible.

Reply:
I'm not convinced, but this is surely no blocker ;-)